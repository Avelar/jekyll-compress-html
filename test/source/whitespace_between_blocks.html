---
layout: compress
---

<ul> <li>One</li>  <li>Two</li>   <li>Three</li></ul>
