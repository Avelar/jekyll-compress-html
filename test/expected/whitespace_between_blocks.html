<div><div>One</div><div>Two</div><div>Three</div></div>
